openqa_review/openqa_review.py

```diff
@@ -1033,19 +1043,31 @@ def _todo_issues_str(self):
                 + self.issues["existing"]["todo"],
                 self.args.show_empty,
             )
-        todo_issues = todo_review_template.substitute(
+        if not self.has_todo_issues():
+            return ""
+
+        if self.args.todo_only:
+            todo_issues = issue_listing("**new**", self.issues["new"]["todo"], self.args.show_empty, no_headers=True)
+            todo_issues += issue_listing("**existing**", self.issues["existing"]["todo"], self.args.show_empty, no_headers=True)
+            return todo_issues
+        return todo_review_template.substitute(
             {
                 "new_issues": issue_listing("***new issues***", self.issues["new"]["todo"], self.args.show_empty),
                 "existing_issues": issue_listing(
                     "***existing issues***", self.issues["existing"]["todo"], self.args.show_empty
                 ),
             }
         )
-        return todo_issues if (self.issues["new"]["todo"] or self.issues["existing"]["todo"]) else ""
 
     def __str__(self):
         """Return as markdown."""
         abbrev = self.args.abbreviate_test_issues
+        if self.args.todo_only:
+            if not self.has_todo_issues():
+                return ""
+
+            return self._todo_issues_str()
```

Comment:
how about

```suggestion
            return self._todo_issues_str() if self.has_todo_issues() else ""
```

or at least

```suggestion
            if self.has_todo_issues():
                self._todo_issues_str()    
            return ""
```

Reply:
done